CMakeLists.txt

```diff
@@ -164,6 +164,9 @@ ENDIF()
 IF((CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX) AND NOT CMAKE_COMPILER_IS_MINGW)
   IF (BUILD_SHARED_LIBS AND CMAKE_SIZEOF_VOID_P EQUAL 8) # -fPIC is only required for shared libs on 64 bit
      SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+     IF(ASSIMP_BUILD_ZLIB)
+	     SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
```

Comment:
That's right in principle, but it will make Travis builds take even longer and be more vulnerable to flaky failures :-( I'd say, let's just always specify FPIC. Yes, it can blow up binary sizes but I also think that it won't matter much for the amount of C code we compile.